hoomd/md/methods/methods.py

```diff
@@ -753,7 +753,7 @@ class Langevin(Method):
 
         gamma_r (TypeParameter[``particle type``,[`float`, `float` , `float`]]):
             The rotational drag coefficient tensor for each particle type
-            :math:`[\mathrm{time}^{-1}]`.
+            :math:`[\mathrm{mass} \cdot \mathrm{length}^{2} \cdot \mathrm{time}^{-1}]`.
```

Comment:
This change is good, make sure to also change the units in the parameter description at the top of the docstring for `default_gamma_r`.